Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/Automation/DataGridCellAutomationPeer.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// #define DEBUG_AUTOMATION
```

Comment:
I would remove these comments from the headers. Doesn't it make more sense to turn these on/off project-wise thru the csproj's DefineConstants?

Reply:
Fair enough. Done. Thanks.